tensorflow/core/kernels/gather_functor_batched_gpu.cu.h

```diff
@@ -29,10 +29,11 @@ namespace tensorflow {
 
 typedef Eigen::GpuDevice GPUDevice;
 
-template <typename T, typename Index,
-          bool is_axis_zero, bool is_batch_dims_zero>
-__global__ void GatherOpKernel(const T* params, const Index* indices, T* out,
-                               int64 outer_size,
+template <typename T, typename Index, bool is_axis_zero,
```

Comment:
Can you please rename `T` to make it more obvious that it isn't necessarily a primitive type?